### PR TITLE
feat(cli/lsp): Sort repl completions

### LINKED ISSUE #15160: Sort registry completions in the repl
Not sure where the sorting should occur, but it would be nice if this was sorted in the REPL:

image

### PR DESCRIPTION
**Pull request by 2shiori17**
fixes #15160

### CODE REVIEW

File: cli/lsp/repl.rs [hunk 1]
```diff
@@ -157,10 +160,19 @@ impl ReplLanguageServer {
           CompletionTextEdit::Edit(edit) => Some(ReplCompletionItem {
             new_text: edit.new_text,
             range: lsp_range_to_std_range(&text_info, &edit.range),
+            sort_text: item.sort_text,
+            label: item.label,
           }),
           CompletionTextEdit::InsertAndReplace(_) => None,
         })
       })
+      .sorted_by_key(|item| {
```
Comment (dsherret) [inline cli/lsp/repl.rs:169]: I think we could just sort `items` vector in place above? That would allow us to remove the added properties and dependency.

**Review by dsherret — CHANGES_REQUESTED** [review]
Thanks, but I think we can just sort the `items` vector in place without adding a dependency.

File: cli/lsp/repl.rs (hunk 1 above)
Comment (2shiori17) [reply cli/lsp/repl.rs]: It's exactly as you said. I fixed it.

**Review by dsherret — APPROVED** [review]
LGTM. Thanks @2shiori17!

**Pull request merged**